### ISSUE TITLE
Fix attribute spelling error in ModalBox & Popover docs

### ISSUE THREAD

**Issue by andybraren** [patternfly/patternfly-next#590]
While working on #525 I noticed the component docs refer to "aria-labeled" with one L instead of two. The attribute is spelled correctly in the examples, but the docs should be updated to match.
